packages/app-project/src/screens/ProjectHomePage/components/MessageFromResearcher/MessageFromResearcher.js

```diff
@@ -0,0 +1,71 @@
+import { Markdownz, SpacedText } from '@zooniverse/react-components'
+import counterpart from 'counterpart'
+import { Box, Paragraph, Image } from 'grommet'
+import { string } from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import en from './locales/en'
+import ContentBox from '../../../../shared/components/ContentBox'
+
+counterpart.registerTranslations('en', en)
+
+const StyledParagraph = styled(Paragraph)`
+  max-width: 100%;
+`
+
+const StyledAvatar = styled(Image)`
+  width: 75px;
+  height: 75px;
+  object-fit: cover;
+  border-radius: 50%;
+`
+
+counterpart.registerTranslations('en', en)
+
+const components = {
+  p: (nodeProps) => (
+    <Paragraph
+      children={nodeProps.children}
+      margin={{ top: 'none' }}
+      size='xlarge'
+    />
```

Comment:
Out of interest, why isn't this written as:
```
<Paragraph>
  {nodeProps.children}
</Paragraph>
```

Reply:
No special reason, other than if it turns into a list I think this way is easier to read